test/test-cases/options-exportEmptyLocals-false/source.css

```diff
@@ -0,0 +1,20 @@
+:local(.layer1A) {
+  color: red;
+}
+
+:local(.layer2A) {
+  composes: layer1A;
+}
+
+:local(.layer1B) {
+}
```

Comment:
Let's add couple tests:
- comment(s) inside block
- comment(s) inside selector before/after
- invalid composes (i.e. not reference)

Reply:
Sounds good. Do you separate test cases or should everything go in these existing files?

Reply:
Let's add them here

Reply:
I've now force-pushed changes. I had to add the invalid composes in a separate directory, but the other tests were added to the original files. Not entirely sure if I got all your suggestions for tests correctly, but please have a look and let me know.

Some comments are removed, but that's what the current code does as well and I've made the tests reflect that. Since I added the `true` or "default" tests in a separate commit before the other changes, you can check out that commit and run `yarn test` to confirm it if you wish. You can also see that only the `:export` blocks differ:
```sh
diff test/test-cases/options-exportEmptyLocals-{true,false}/expected.css
```

Also, I should mention that I had overlooked the fact that comments can go inside the declaration blocks, so it was good that you pointed that out and I've updated the code accordingly. Let me know if there are any other special cases that I'm unaware of.